front_materials/acknowledgments.tex

```diff
@@ -1 +1,6 @@
-Put your acknowledgements text here.
\ No newline at end of file
+% Suhak Lee: no indentation applied in Acknowledgements chapter. So I had to forcefully indent using \hspace{15}
+
+\hspace{15pt} Put your acknowledgements text here.
+
+\hspace{15pt} Another paragraph
+
```

Comment:
Can you implement the paragraph indentation in the thesis-umich.cls file instead of here please? That'll make it easier for anyone who isn't as familiar with latex, and harder for them to erase your change. I'd recommend using something like the \indent command instead of an \hspace. Not sure exactly which is correct (I'm not *that* good at latex myself), but something like that.